task-list-rails/app/controllers/tasks_controller.rb

```diff
@@ -27,9 +27,32 @@ def destroy
     redirect_to "/"
   end
 
+  def edit
+    id = params[:id]
+    @task = Task.find(id)
+  end
+
+  def update
+    Task.update(params[:id],
```

Comment:
Because you have the `task_params` method already, using `params.permit` you can actually simplify this line. Instead, you could write `Task.update(params[:id], task_params[:task])`.